patch from upstream ports repo

From 79abd1fbfcd082e5e9857d41d0acb4f64e9299de Mon Sep 17 00:00:00 2001
From: Aurelien Jarno <aurelien@aurel32.net>
Date: Tue, 1 Mar 2011 10:02:04 +1000
Subject: [PATCH] alpha: Add f_flags to struct statfs

---
 ChangeLog.alpha                             |   11 ++++++++---
 sysdeps/unix/sysv/linux/alpha/bits/statfs.h |    6 ++++--
 2 files changed, 12 insertions(+), 5 deletions(-)

2011-02-28  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/alpha/bits/statfs.h (struct statfs,
	struct statfs64): Add f_flags field.

diff --git a/sysdeps/unix/sysv/linux/alpha/bits/statfs.h b/sysdeps/unix/sysv/linux/alpha/bits/statfs.h
index d838e6b..157591d 100644
--- ports/sysdeps/unix/sysv/linux/alpha/bits/statfs.h
+++ ports/sysdeps/unix/sysv/linux/alpha/bits/statfs.h
@@ -42,7 +42,8 @@ struct statfs
     __fsid_t f_fsid;
     int f_namelen;
     int f_frsize;
-    int f_spare[5];
+    int f_flags;
+    int f_spare[4];
   };
 
 #ifdef __USE_LARGEFILE64
@@ -58,7 +59,8 @@ struct statfs64
     __fsid_t f_fsid;
     int f_namelen;
     int f_frsize;
-    int f_spare[5];
+    int f_flags;
+    int f_spare[4];
   };
 #endif
 
-- 
1.7.4.1

